openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/storages/MultiTreeSPEntry.java

```diff
@@ -20,13 +20,16 @@
  */
 public class MultiTreeSPEntry implements Comparable<MultiTreeSPEntry> {
 
-	private int adjNode;
-	private boolean visited = false;
-	private MultiTreeSPEntryItem[] items;
+	protected int adjNode;
+	protected int edge;
+	protected int originalEdge;
```

Comment:
Is `originalEdge` actually necessary here and ever different from `edge`?

![image](https://user-images.githubusercontent.com/6545356/134662600-333d5630-6730-48be-837c-f82fa901fbdb.png)

Reply:
Sorry, this is a reused class that I did not implement. As far as I can tell, it is never necessary (and even wrong, as the original edge can differ for different subItems), but it was not the focus of this development to change existing classes.

Reply:
Now I'm confused as you seem to contradict yourself. Your PR does change `MultiTreeSPEntry` by adding`originalEdge` which to me appears obsolete.

Reply:
Sorry, you are completely right. It was a leftover from my dev work.

Reply:
No worries, thanks for sorting this out!